AlbumList/AlbumListViewController.swift

```diff
@@ -0,0 +1,187 @@
+    
+    private func APIAlbums(completed: @escaping () -> ()) {
+        guard let url = URL(string: "https://jsonplaceholder.typicode.com/albums") else {
+            // TODO: Error handling - not valid URL
+            return
+        }
+
+        var request = URLRequest(url: url)
+        request.httpMethod = "GET"
+        request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+
+        let session = URLSession.shared
+        let task = session.dataTask(with: request, completionHandler: { data, response, error -> Void in
+            guard let httpResponse = response as? HTTPURLResponse else {
+                // TODO: Error handling - not valid response
+                return
+            }
+            
+            switch httpResponse.statusCode {
+            case 200:
+                guard let data = data else {
+                    // TODO: Error handling
+                    return
+                }
+                let jsonDecoder = JSONDecoder()
+                let responseModel = try? jsonDecoder.decode([Album].self, from: data)
+                self.albums = responseModel!
+                
+                DispatchQueue.main.async {
+                    completed()
+                }
+                
+            default:
+                // TODO: Error handling - invalid response
+                return
+            }
+        })
+
+        task.resume()
+    }
+    
+    //MARK: - USER API call
+    
+    private func APIUsers() {
+        guard let url = URL(string: "https://jsonplaceholder.typicode.com/users") else {
+            // TODO: Error handling - not valid URL
+            return
+        }
+
+        var request = URLRequest(url: url)
+        request.httpMethod = "GET"
+        request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+
+        let session = URLSession.shared
+        let task = session.dataTask(with: request, completionHandler: { data, response, error -> Void in
+            guard let httpResponse = response as? HTTPURLResponse else {
+                // TODO: Error handling - not valid response
+                return
+            }
+            
+            switch httpResponse.statusCode {
+            case 200:
+                guard let data = data else {
+                    // TODO: Error handling
+                    return
+                }
+                
+                let jsonDecoder = JSONDecoder()
+                let userResponseModel: [User]? = try? jsonDecoder.decode([User].self, from: data)
+                self.users = userResponseModel ?? []
+//                DispatchQueue.main.async {
+//                    completed()
+//                }
+                print("User API Called")
+            default:
+                // TODO: Error handling - invalid response
+                return
+            }
+        })
+
+        task.resume()
+    }
+}
+
+//MARK: - Structs
+
+public struct Album: Codable {
```

Comment:
Lets move these to a separate swift file. So you can have Album.swift and User.swift